integration-tests/src/test/java/tachyon/shell/command/LsThenloadMetadataTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.conf.TachyonConf;
+import tachyon.exception.ExceptionMessage;
+import tachyon.exception.TachyonException;
+import tachyon.shell.AbstractTfsShellTest;
+import tachyon.util.UnderFileSystemUtils;
+import tachyon.util.io.PathUtils;
+
+/**
+ * Test for ls with a wildcard argument
+ */
+public class LsThenloadMetadataTest extends AbstractTfsShellTest {
```

Comment:
I think you group this and `LsWildcardTest` into `LsTest`. Basically each command has its own related tests.